sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClient.java

```diff
@@ -326,6 +328,38 @@ public String getEntityPath() {
         return scheduleMessageInternal(message, scheduledEnqueueTime, null);
     }
 
+    /**
+     * Sends a scheduled messages to the Azure Service Bus entity this sender is connected to. A scheduled message is
+     * enqueued and made available to receivers only at the scheduled enqueue time.
+     *
+     * @param messages Message to be sent to the Service Bus Queue.
+     * @param scheduledEnqueueTime OffsetDateTime at which the message should appear in the Service Bus queue or topic.
+     *
+     * @return The sequence number of the scheduled message which can be used to cancel the scheduling of the message.
+     *
+     * @throws NullPointerException if {@code messages} or {@code scheduledEnqueueTime} is {@code null}.
+     */
+    public Flux<Long> scheduleMessages(Iterable<ServiceBusMessage> messages, OffsetDateTime scheduledEnqueueTime) {
+        return scheduleMessagesInternal(messages, scheduledEnqueueTime, null);
+    }
+
+    /**
+     * Sends a scheduled messages to the Azure Service Bus entity this sender is connected to. A scheduled message is
+     * enqueued and made available to receivers only at the scheduled enqueue time.
+     *
+     * @param messages Message to be sent to the Service Bus Queue.
+     * @param scheduledEnqueueTime Instant at which the message should appear in the Service Bus queue or topic.
+     * @param transactionContext to be set on batch message before scheduling them on Service Bus.
+     *
+     * @return The sequence number of the scheduled message which can be used to cancel the scheduling of the message.
+     *
+     * @throws NullPointerException if {@code message} or {@code scheduledEnqueueTime} is {@code null}.
+     */
+    public Flux<Long> scheduleMessages(Iterable<ServiceBusMessage> messages, OffsetDateTime scheduledEnqueueTime,
+        ServiceBusTransactionContext transactionContext) {
+        return scheduleMessagesInternal(messages, scheduledEnqueueTime, transactionContext);
```

Comment:
You don't need this internal method at all. The overload on top can call this method with transaction context as null.